libsplinter/Cargo.toml

```diff
@@ -128,17 +129,18 @@ authorization-handler-allow-keys = ["authorization"]
 authorization-handler-maintenance = ["authorization"]
 authorization = ["rest-api"]
 authorization-handler-rbac = ["authorization"]
-biome-credentials = ["bcrypt"]
-biome-key-management = []
-biome-notifications = []
+biome = []
```

Comment:
May be useful to comment on this feature being internal - it doesn't provide any functionality on its own

Reply:
But... if that changes, do we need to remember to update the comment here? This is conceptually similar to admin-service, registry, and some of the transact features which just enable the parent module.